docs/Guides/Advertising/connectedAccount.md

```diff
@@ -0,0 +1,262 @@
+---
+tags: [Advertising Intelligence, adintel, account, connectedAccount]
+---
+# Getting Advertising Connected Accounts
+
+A `Connected Account` represents the metadata of an advertising account that has been connected to a specific Business Location in Advertising Intelligence.  
+
+> For info on Business Locations see the [Businesses guide](../Accounts.md)
+
+## Setup
+
+Create an access token with `adintel` scopes following the [Authorization guide](../../Authorization/Authorization.md).
+
+## Examples
+
+### Retrieving A Connected Account With ID
+
+The ID of a connected account is composite of the following parts, separated by `:`
+1. The Business Location ID
+1. Provider of the account, either `google`, `facebook` or `localAds`
+1. The assigned account ID
+For example, an account with ID `1324354698` from Google Adwords that's connected for business `AG-X5FZQG6T25` would have its ID to be `AG-X5FZQG6T25:google:1324354698`
+
+A connected account can be retrieved using its ID.  This data can be used to provide an overview of a single advertising account for a business when the advertising account ID is known.
```

Comment:
Instead of telling the client how to build an ID. Tell them to search for it using filters on the list endpoint. You can swap the order of these examples since get one builds on list.

```suggestion
A single connected account can be retrieved using the ID previously found using the list operation.  This data can be used to provide an overview of a single advertising account for a business when the advertising account ID is known.
```